openapi.json

```diff
@@ -2363,6 +2395,125 @@
         }
       }
     },
+    "/users/{id}/category_rules": {
+      "get": {
+        "summary": "List category rules in user",
+        "description": "Lists all category rules belonging to a user by their ID.",
+        "tags": [
+          "Category Rules"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique identifier of the user.",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 42
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/CategoryRule"
+                  }
+                }
+              }
+            }
+          },
+          "403": {
+            "$ref": "#/components/responses/403"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/categories/{id}/category_rules": {
+      "post": {
+        "summary": "Create category rule in category",
+        "description": "Creates a rule to allocate a category to transactions",
+        "tags": [
+          "Category Rules"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique identifier of the category.",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 42
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "required": [
+                  "category_id",
+                  "payee_matches"
+                ],
+                "properties": {
+                  "category_id": {
+                    "type": "integer",
+                    "description": "The ID of the category the rule is for.",
+                    "example": 42
+                  },
+                  "payee_matches": {
+                    "type": "string",
+                    "description": "The keyword/s to match the transaction payees",
+                    "example": "Countdown"
+                  },
+                  "apply_to_uncategorised": {
+                    "type": "boolean",
+                    "description": "Apply the category to all uncategorised transactions matching the keyword/s.",
+                    "example": true
+                  },
+                  "apply_to_all": {
+                    "type": "boolean",
+                    "description": "Apply the category to all transactions matching the keyword/s.",
```

Comment:
Same here - this specifically applies **only** the newly created rule, not all the rules that the category has.